pom.xml

```diff
@@ -103,6 +103,24 @@
             <version>2.14</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-cps</artifactId>
+            <version>2.39</version>
+            <scope>compile</scope>
```

Comment:
Why do you use the `compile` scope? You should be using the default scope or `test` for 99% of the use-cases for plugins

Reply:
Actually, it was already placed when I created the skeleton.

Reply:
should I remove the compile scope ?